### PR TITLE
Added the IPAM feature description

### PR DESCRIPTION
**Pull request by veenamj-avesha**
Added the IPAM feature description, which is already added as part of the enterprise docs.

### CODE REVIEW

File: versioned_docs/version-0.6.0/features/ipam.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# IP Address Management 
+
+IP Address Management (IPAM) is a method of planning, tracking, and managing the IP address space 
+used in a network. The `maxClusters` parameter of the slice configuration represents IPAM.
```
Comment (mnmaha) [inline versioned_docs/version-0.6.0/features/ipam.mdx:4]: it doesn't represent IPAM, it helps with it.. can we rephrase

File: versioned_docs/version-0.6.0/features/ipam.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+used in a network. The `maxClusters` parameter of the slice configuration represents IPAM. 
+
+This parameter sets the maximum number of worker clusters that you can connect to a slice. The 
+maximum number of worker clusters affects the subnet calculation of a worker cluster. The subnet 
+in turn determines the number of host addresses a worker cluster gets for its application pods. 
+
+For example, if the slice subnet is 10.1.0.0/16 and the maximum number of clusters is 16, then each 
+cluster gets a subnet of 10.1.x.0/20, where x = 0, 16, or 32.
+
+This is a significant parameter that is only configured during a slice creation. If this parameter is
```
Comment (mnmaha) [inline versioned_docs/version-0.6.0/features/ipam.mdx:13]: This is a significant parameter that can only be configured during ...

File: versioned_docs/version-0.6.0/features/ipam.mdx (hunk 1 above)
Comment (veenamj-avesha) [reply versioned_docs/version-0.6.0/features/ipam.mdx:4]: rephrased it as `helps with IPAM`

File: versioned_docs/version-0.6.0/features/ipam.mdx (hunk 2 above)
Comment (veenamj-avesha) [reply versioned_docs/version-0.6.0/features/ipam.mdx:13]: rephrased

File: versioned_docs/version-0.6.0/features/ipam.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+used in a network. The `maxClusters` parameter of the slice configuration helps with IPAM. 
+
+This parameter sets the maximum number of worker clusters that you can connect to a slice. The 
+maximum number of worker clusters affects the subnet calculation of a worker cluster. The subnet 
+in turn determines the number of host addresses a worker cluster gets for its application pods. 
+
+For example, if the slice subnet is 10.1.0.0/16 and the maximum number of clusters is 16, then each 
+cluster gets a subnet of 10.1.x.0/20, where x = 0, 16, or 32.
+
+This is a significant parameter that can only be configured during a slice creation. If this parameter is
```
Comment (uma-kt) [inline versioned_docs/version-0.6.0/features/ipam.mdx:13]: during a slice creation -> The  `a` article is not required

**Review by uma-kt — APPROVED** [review]
LGTM

**Pull request merged**